### PR TITLE
adding file type to cache

### PR DESCRIPTION
**Pull request by ThaDafinser**
Follow up from #148

### CODE REVIEW

File: src/Data/PropertyHolder.php [hunk 1]
```diff
@@ -21,9 +21,10 @@
  * THE SOFTWARE.
  *
  * @category   Browscap-PHP
- * @package    Data
```
Comment (ThaDafinser) [inline src/Data/PropertyHolder.php]: This was removed by `vendor/bin/php-cs-fixer fix src` ....

File: src/Data/PropertyHolder.php [hunk 2]
```diff
@@ -31,25 +32,27 @@
 namespace BrowscapPHP\Data;
 
 /**
- * Class PropertyHolder
+ * Class PropertyHolder.
  *
  * @category   Browscap
- * @package    Data
```
Comment (ThaDafinser) [inline src/Data/PropertyHolder.php]: This was removed by `vendor/bin/php-cs-fixer fix src` ....

File: src/Helper/Exception.php [hunk 3]
```diff
@@ -21,9 +21,10 @@
  * THE SOFTWARE.
  *
  * @category   Browscap-PHP
- * @package    Helper
```
Comment (ThaDafinser) [inline src/Helper/Exception.php]: This was removed by `vendor/bin/php-cs-fixer fix src` ....

File: src/Data/PropertyHolder.php (hunk 1 above)
Comment (asgrim) [inline src/Data/PropertyHolder.php]: That's fine, they're pretty much not needed anyway :)

**Comment by ThaDafinser** [comment]
@asgrim the tests are fine now.

As soon as this is ready i will provide 3 different adapters here based on the type, which will check this type.
https://github.com/ThaDafinser/UserAgentParser/tree/feature/browscap/src/Provider

**Comment by asgrim** [comment]
Thanks @ThaDafinser (FYI, I'm pretty sure we're almost ready for 3.0.0 release)

**Pull request merged**